apps/pie-docs/src/foundations/iconography/code.md

```diff
@@ -0,0 +1,277 @@
+---
+eleventyNavigation:
+    key: Code
+    parent: Iconography
+    order: 3
+---
+
+## Overview
+- [Our packages](#our-packages)
+- [Pie Icons](#pie-icons)
+  - [Usage - Client-side Javascript](#usage-client-side-javascript)
+    - [Installation](#installation)
+    - [Include the package](#include-the-package)
+    - [HTML](#html)
+    - [Invoke the package](#invoke-the-package)
+  - [Usage - NodeJS](#usage-nodejs)
+    - [Installation](#installation-1)
+    - [Require the package](#require-the-package)
+    - [Call the APIs](#call-the-apis)
+- [React](#react)
+  - [Usage](#usage)
+    - [Installation](#installation-2)
+    - [Import into your project](#import-into-your-project)
+- [Vue](#vue)
+  - [Usage](#usage-1)
+    - [Installation](#installation-3)
+    - [Import into your project](#import-into-your-project-1)
+- [Web Components](#web-components)
+  - [Usage - In Lit Components](#usage-in-lit-components)
+    - [Installation](#installation-4)
+    - [Import into your project](#import-into-your-project-2)
+
+
+---
+
+## Our packages
+
+We have a number of different icon packages available for use in our applications. We have a package for each framework we support, as well as a base package for the icons themselves.
+
+---
+
+## Pie Icons
```

Comment:
I would use the package names for the titles for these sections, either just as is, or in brackets.

```md
## `pie-icons` 
```

or

```md
## PIE Icons (`pie-icons`)
```